src/Directory.Build.props

```diff
@@ -18,6 +18,7 @@
     <GenerateTestProjection Condition="'$(GenerateTestProjection)' == ''">false</GenerateTestProjection>
     <CsWinRTGenerateProjection>$(GenerateTestProjection)</CsWinRTGenerateProjection>
     <CsWinRTWindowsMetadata Condition="'$(CsWinRTWindowsMetadata)' == ''">$(WindowsSDKVersion.TrimEnd('\'))</CsWinRTWindowsMetadata>
+    <CsWinRTWindowsMetadata Condition="'$(CsWinRTWindowsMetadata)' == ''">sdk</CsWinRTWindowsMetadata>
```

Comment:
Is this only possible if `WindowsSDKVersion` was empty?

Reply:
Right, I believe WindowsSDKVersion is only set in a VS developer command prompt.  And outside of it, it doesn't seem to be set.  So I had to fallback to our logic to handle this.